src/components/Accordion/Accordion.jsx

```diff
@@ -47,9 +92,19 @@ Accordion.propTypes = {
     ]).isRequired,
 
     /**
-     * AccordionItem's id to set an item open by default
+     * AccordionItem's id opened by default (isMultipleOpenable ? Array : String)
+     */
+    defaultOpenedItemId: idPropType,
+
+    /**
+     * AccordionItem's id open (isMultipleOpenable ? Array : String)
+     */
+    openedItemId: idPropType,
```

Comment:
Should also expose setOpenedItemId to control the component, if openedItemId is set